cmd/repo-init/main.go

```diff
@@ -211,19 +212,27 @@ A test named %s already exists. Please choose a different name.\n`, test.As)
 					break
 				}
 			}
+
+			clusterProfiles := sets.NewString("gcp", "aws", "azure4")
 			test.Profile = api.ClusterProfile(fetchOrDefaultWithPrompt("Which specific cloud provider does the test require, if any? ", string(api.ClusterProfileAWS)))
 			for {
-				if test.Profile.ClusterType() == "" {
+				if !clusterProfiles.Has(string(test.Profile)) {
 					fmt.Printf(`
-No cluster profile named %s exists. Please choose one from: %v.\n`, test.Profile, api.ClusterProfiles())
+Cluster profile %s is not valid. Please choose one from: gcp, aws, azure4.\n`, test.Profile)
```

Comment:
It'd be nice to DRY this, if we ever give the users more options we'll immediately forget to update this string. Let's have a canonical slice and infer the set (above) and the string from it?